globus_automate_client/action_client.py

```diff
@@ -84,8 +88,9 @@ def run(
         :param force_path: A URL to use for running this action, ignoring any
             previous configuration
         :param label: Set a label for the Action that is run.
-        :param run_monitors: May be used as an alias for ``monitor_by``
-        :param run_managers: May be used as an alias for ``manage_by``
+
+        ``run_monitors`` may be used as an alias for ``monitor_by``,
+        and ``run_managers`` may be used as an alias for ``manage_by``.
```

Comment:
These methods are actually documented on our client doc site and the `:param:` format for the docstring is how Sphinx knows to parse these and render them on readthedocs.

Let's keep the pre-existing to make sure our docs stay in sync with the code.

Reply:
I understand that the `:param:` format is used by Sphinx and autodoc. I deliberately changed these because neither are actual parameters anymore and appeared to be deprecated in favor of `monitor_by` and `manage_by`. I wanted to reduce their visibility so new code is less likely to depend on these fallback aliases.

I'll move it back.

Reply:
Done.

Reply:
Oh I see. The deprecated behavior is actually reverse -- the new terminology is `run_monitors` and `run_managers` and we're trying to migrate folks off of `monitor_by` and `manage_by`. Sorry for causing some flipflop here.